openmc/data/data.py

```diff
@@ -273,6 +277,34 @@ def atomic_weight(element):
                          .format(element))
 
 
+def half_life(isotope):
+    """Return half-life of isotope in seconds or None if isotope is stable
+
+    Half-life values are from the `ENDF/B-VIII.0 decay sublibrary
+    <https://www.nndc.bnl.gov/endf-b8.0/download.html>`_.
+
+    Parameters
+    ----------
+    isotope : str
+        Name of isotope, e.g., 'Pu239'
+
+    Returns
+    -------
+    float
+        Half-life of isotope in [s]
+
+    """
+    global _HALF_LIFE
+    if not _HALF_LIFE:
+        # Load ENDF/B-VIII.0 data from JSON file
+        half_life_path = Path(__file__).with_name('half_life.json')
+        _HALF_LIFE = json.load(half_life_path.read_text())
```

Comment:
Oops, this should fix the failing tests:
```suggestion
        _HALF_LIFE = json.loads(half_life_path.read_text())
```